development/verify-locale-strings.js

```diff
@@ -180,6 +180,7 @@ async function verifyEnglishLocale() {
   const javascriptFiles = await glob(
     [
```

Comment:
I wonder if this array is worth moving to a constant at the top. I guess this is fine for now though.